rust/lance/src/index/vector/ivf/v2.rs

```diff
@@ -647,6 +676,7 @@ mod tests {
     #[case(4, DistanceType::L2, 1.0)]
     #[case(4, DistanceType::Cosine, 1.0)]
     #[case(4, DistanceType::Dot, 1.0)]
+    #[case(4, DistanceType::Hamming, 0.9)]
```

Comment:
even it's flat search, the recall for hamming can't always hit 100%, because there are many vectors that are at the same distance from the query vector